src/vm/analysis/errors.rs

```diff
@@ -55,6 +55,7 @@ pub enum CheckErrors {
     PublicFunctionMustReturnResponse(TypeSignature),
     DefineVariableBadSignature,
     ReturnTypesMustMatch(TypeSignature, TypeSignature),
+    CyclingDependencies(Vec<String>),
```

Comment:
Just a suggestion -- a clearer name for this error would be "DependencyCycle" or "CircularReference"

Reply:
Addressed in https://github.com/blockstack/blockstack-core/pull/1079/commits/fd23f90f92281830fb2b4d2708ea01a041e66cf0